scipy/optimize/_tstutils.py

```diff
@@ -753,14 +753,14 @@ def fun9(x):
     [fun2, [1e-12, 1e12], fun2.root, 48],
     [fun3, [0, 5], fun3.root, 21],
     [fun3, [-10, 10], fun3.root, 23],
-    [fun3, [1e-4, 1e4], fun3.root, 36],
-    [fun3, [1e-6, 1e6], fun3.root, 45],
-    [fun3, [1e-10, 1e10], fun3.root, 55],
+    [fun3, [-1e4, 1e4], fun3.root, 36],
```

Comment:
These are corrections according to Table 2 of [1]. Note that the last column is the number of iterations reported in [1]. `test_nit_expected` checks that the number of iterations required by `_chandrupatla` is identical. This provides evidence that the algorithm is implemented faithfully.